opm/grid/cpgrid/CpGrid.cpp

```diff
@@ -2079,19 +2083,20 @@ void CpGrid::addLgrsUpdateLeafView(const std::vector<std::array<int,3>>& cells_p
 
     int non_empty_lgrs = 0;
     for (int level = 0; level < static_cast<int>(startIJK_vec.size()); ++level) {
-        if (lgrs_with_at_least_one_active_cell[level] == 0) {
+        // Do not throw if all cells of an LGR are inactive in a parallel run (The process might not 'see' those cells.)
+        if ((lgrs_with_at_least_one_active_cell[level] == 0) && distributed_data_.empty()) {
```

Comment:
I think there are two options here:
1. We never throw for empty LGRs, but just issue a warning via OpmLog
2. Also do the check in parallel.

In both cases we will need to do a sum of the cells of each lgr on all processes:
```
auto localLgrCells = countActiveCellsInLGR();
auto globalLgrCells = comm_.sum(localLgrCells);
if(globalLgrCells==0) ....
```

Reply:
Now, we never throw for empty LGRs and issue a warning via Opm::log. The check is done in parallel, in the sense that, each process sees only its own cells, and then a process could conclude that an LGR is empty, when for the global view it's not empty (it contains active cells not seen by the current process).

Reply:
Now, we also have such a check when global lgrs only contain inactive cells.